test/resources/language/pug.pug

```diff
@@ -1,6 +1,6 @@
 doctype html
  html
   head
-    title Jade Examples
+    title Pug Examples
```

Comment:
Please move this file into the `pug/` directory to put it beneath all the other pug files.